Added documentation for nina (#19472)
home-assistant/home-assistant.github.io: source/_integrations/nina.markdown

---
title: NINA
description: Instructions on how to set up NINA warnings in Home Assistant.
ha_category:
  - Binary Sensor

> ha_release: 2021.11.0

Reviewer: ```suggestion
ha_release: 2021.2
```

Reviewer 2: This should be 

```suggestion
ha_release: 2022.2
```

> ha_release: 2021.11.0

Reviewer 2: ```suggestion
ha_release: 2022.1.0
```

Reviewer: This suggestion is incorrect. We don't prefix the minor part of releases with a leading `0`, additional, there will not be a `2022.1` release, as the first upcoming release will be `2022.2`.

Reviewer 2: Lets just use your suggestion then :+1:

ha_iot_class: Cloud Polling
ha_config_flow: true
ha_codeowners:
  - '@DeerMaximum'
ha_domain: nina
ha_platforms:
  - binary_sensor
---

> The [`nina`](https://www.bbk.bund.de/DE/Warnung-Vorsorge/Warn-App-NINA/warn-app-nina_node.html) integration displays warnings from the [`Bundesamtes für Bevölkerungsschutz und Katastrophenhilfe`](https://www.bbk.bund.de/) in Germany.

Reviewer: ```suggestion
The [NINA](https://www.bbk.bund.de/DE/Warnung-Vorsorge/Warn-App-NINA/warn-app-nina_node.html) integration displays warnings from the [Bundesamtes für Bevölkerungsschutz und Katastrophenhilfe](https://www.bbk.bund.de/) in Germany.
```

> The [`nina`](https://www.bbk.bund.de/DE/Warnung-Vorsorge/Warn-App-NINA/warn-app-nina_node.html) integration displays warnings from the [`Bundesamtes für Bevölkerungsschutz und Katastrophenhilfe`](https://www.bbk.bund.de/) in Germany.

Reviewer 2: ```suggestion
The [NINA](https://www.bbk.bund.de/DE/Warnung-Vorsorge/Warn-App-NINA/warn-app-nina_node.html) integration displays warnings from the [Bundesamt für Bevölkerungsschutz und Katastrophenhilfe](https://www.bbk.bund.de/) in Germany.
```

> For each county/city its create warning slots which change to Unsafe when warnings are present. The text of the warning and the metadata are stored in the attributes of the slots.

Reviewer 2: ```suggestion
For each county/city it creates warning slots which change to Unsafe when warnings are present. The text of the warning and the metadata are stored in the attributes of the slots.
```

> For each county/city it creates warning slots which change to Unsafe when warnings are present. The text of the warning and the metadata are stored in the attributes of the slots.

Reviewer: ```suggestion
For each county/city it creates warning slots that change to Unsafe when warnings are present. The text of the warning and the metadata are stored in the attributes of the slots.
```
